README.rst

```diff
@@ -160,74 +165,25 @@ If you want to make your robot speak, you can use the ``Sound.speak`` method:
   sound = Sound()
   sound.speak('Welcome to the E V 3 dev project!')
 
-Make sure to check out the `User Resources`_ section for more detailed
-information on these features and many others.
-
 Using Micropython
 -----------------
 
-Normal Python too slow? Try `Micropython`_ if it supports the features your
-project needs.
-
-User Resources
---------------
+Normal Python too slow? Review `Micropython`_ to see if it supports the
+features your project needs.
 
 Library Documentation
-    **Class documentation for this library can be found on** `our Read the Docs page`_ **.**
-    You can always go there to get information on how you can use this
-    library's functionality.
-
-Demo Code
-    There are several demo programs that you can run to get acquainted with
-    this language binding. The programs are available at
-    https://github.com/ev3dev/ev3dev-lang-python-demo
+---------------------
 
-ev3python.com
-    One of our community members, @ndward, has put together a great website
-    with detailed guides on using this library which are targeted at beginners.
-    If you are just getting started with programming, we highly recommend
-    that you check it out at `ev3python.com`_!
+**Class documentation for this library can be found on
+** `our Read the Docs page`_ **.** You can always go there to get
```

Comment:
Looking at the preview, it doesn't seem like the formatting is working... I don't know why though. Might just be GitHub's diff viewer.

Reply:
I preview and validate the rst formatting with the reStructuredText VS Code extension... not sure if GitHub offers a renderer (like Markdown)